obal/data/modules/srpm.py

```diff
@@ -137,11 +137,16 @@ def main():
         command += [os.path.join(base_dir, '%s.spec' % os.path.basename(package))]
 
         try:
-            result = run_command(command)
+            command_output = run_command(command)
         except subprocess.CalledProcessError as error:
             module.fail_json(msg='Failed to srpm build', command=' '.join(command), output=error.output)
 
-        result = result.split("Wrote: ")[-1].rstrip()
+        result = None
+        for line in command_output.splitlines():
+            if line.startswith("Wrote: "):
+                result = line.split("Wrote: ")[-1].rstrip()
```

Comment:
Should it break after this?

You can then also consider the `else` statement for `for`, though I know plenty of people don't like that.

Reply:
Done. But yes, `for/else` ain't my biggest friend.

Reply:
I had thought you'd move the `module.fail_json` into `else`. Though I now realize that possibly you also wanted to capture empty strings. If you're not moving `fail_json` into `else`, I'd say that your previous version where you first declared `result = None` before the loop was better (but with `break` as an optimization).